.github/workflows/build.yml

```diff
@@ -22,11 +28,20 @@ jobs:
       - name: validate packages
         shell: pixi run bash -e {0}
         run: pixi run -e dev validate
+      - name: upload data artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: data-${{ github.event.pull_request.number }}
+          path: |
+            **/generated/**
```

Comment:
shouldn't this be more specific, i.e., `./data/generated/**`. Some other parts of the repository, such as the (not existing) documentation, sometimes also contain `generated` folders.

Reply:
Generally yes, but this a workaround to trick upload-artifact into obeying the same folder structure as with the zip during a release.